### PR TITLE
feat(sec group): added configurable security group for secondary cluster

### PR DESCRIPTION
**Pull request by crisboarna**
Hi @neilkuan ,

I have raised this PR in hopes of adding extra needed feature to this useful utility you have here.

- added configurable VPC Security Group to secondary region cluster, up to now it would use default security group or error if default VPC is deleted (best practice)
- refactored CfnOutput for `secondRDSClusterArn`, `seconddbInstanceIdentifier` as you already create them in same file, no need to pass it in custom resource, then getAtt from the output. This also causes error as the onUpdate call to custom resource does not return the properties resulting in CloudFormation vendor error.

### CODE REVIEW

**Comment by neilkuan** [comment]
Hello friend!
Thank for this PR. 
Did you already deploy into your account for testing ?!

**Comment by crisboarna** [comment]
Hi, yes I  have already deployed this for testing in my account for a cross region DMS task using this utility. 

I have added another change to make the target cluster parameter group configurable to accommodate extra DB requirements.

**Comment by crisboarna** [comment]
Hi @neilkuan,

I also fixed a logic bug involving the deprecated `PRIVATE` `subnetType` that would result in configuration of `PUBLIC` `subnetType` if you pass in `PRIVATE_WITH_NAT` or `PRIVATE_ISOLATED`

File: src/index.ts [hunk 1]
```diff
@@ -854,6 +851,16 @@ export class GlobalAuroraRDSSlaveInfra extends cdk.Construct {
 
       cdk.Tags.of(this.dbSubnetGroup).add('Name', 'PublicDBSubnetGroup');
       this.dbSubnetGroup.node.addDependency(rdsVpcSecond);
+    } else {
+      const PrivateSubnet = rdsVpcSecond.selectSubnets({ subnetType: ec2.SubnetType.PRIVATE });
```
Comment (neilkuan) [inline src/index.ts:855]: Can you also write testing in [here](https://github.com/neilkuan/cdk-aurora-globaldatabase/blob/master/test/global-rds.test.ts) ?!
Thanks~

File: src/index.ts [hunk 2]
```diff
@@ -518,6 +518,8 @@ export interface GlobalAuroraRDSMasterProps {
 export interface RegionalOptions {
   readonly region: string;
   readonly dbSubnetGroupName?: string;
+  readonly dbClusterpPG?: string;
```
Comment (neilkuan) [inline src/index.ts:521]:
```suggestion
  readonly dbParameterGroup?: string;
```

File: src/index.ts [hunk 3]
```diff
@@ -761,19 +766,21 @@ export class GlobalAuroraRDSMaster extends cdk.Construct {
         ClusterIdentifier: `${stack.stackName.toLowerCase()}-${options.region}`,
         InstanceType: this.rdsInstanceType,
         rdsIsPublic: this.rdsIsPublic,
-        secondRDSClusterArn: `arn:aws:rds:${options.region}:${stack.account}:cluster:${stack.stackName.toLowerCase()}-${options.region}`,
-        seconddbInstanceIdentifier: `${stack.stackName.toLowerCase()}-${options.region}-1`,
+        secondRDSClusterArn,
+        seconddbInstanceIdentifier,
+        securityGroup: options.securityGroupId,
+        dbParameterGroup: options.dbClusterpPG,
```
Comment (neilkuan) [inline src/index.ts:772]:
```suggestion
        dbParameterGroup: options.dbParameterGroup,
```

**Review by neilkuan — CHANGES_REQUESTED** [review]

File: src/index.ts (hunk 1 above)
Comment (crisboarna) [reply src/index.ts:855]: You already have tests at `149` and `159` for private/public subnets.

File: src/index.ts (hunk 3 above)
Comment (crisboarna) [reply src/index.ts]: Updated.

File: src/index.ts (hunk 2 above)
Comment (crisboarna) [reply src/index.ts]: Updated, as you already had that variable name for cluster parameter group in Master stack in same file, I just copied it over for consistency, but I agree using the name I added in the Python `Custom Resource` is more appropriate.

**Comment by neilkuan** [comment]
CI fail, Please yarn build local, and commit API.md file.

**Comment by crisboarna** [comment]
Already done :)

**Review by neilkuan — APPROVED** [review]
LGTM ~
thanks ~

**Pull request merged**